src/Components/Blazor/Mono.WebAssembly.Interop/src/Mono.WebAssembly.Interop.csproj

```diff
@@ -5,8 +5,8 @@
     <Description>Abstractions and features for interop between Mono WebAssembly and JavaScript code.</Description>
     <PackageTags>wasm;javascript;interop</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <IsPackable>true</IsPackable>
-    <IsShipping>true</IsShipping>
+    <IsPackable>false</IsPackable>
+    <IsShipping>false</IsShipping>
```

Comment:
1. Changing `$(IsPackable)` will break the Microsoft.AspNetCore.Blazor.Templates.csproj template. Suggest that template should either be deleted or removed from the project templates meta-package (if it hasn't already).
2. What are we keeping this project for? I don't see references to it other than in Microsoft.AspNetCore.Blazor.Templates.csproj and general infrastructure files e.g. eng/ProjectReferences.props. Suggest cleaning up that file and ComponentsNoDeps.slnf.
3. When `$(IsPackable)` is `false`, changing `$(IsShipping)` mainly controls versions. Does the version matter?

Reply:
The only thing which matters at this point is to make this not ship any more. Are you saying when `IsPackable` is `false` the package will be shipped anyway?

I don't want to do more than this for now as things will be resolved after blazor-wasm will merge into `master` (eventually)

Reply:
> Are you saying when `IsPackable` is `false` the package will be shipped anyway?

No. But, the project has a version which will show up in the Blazor template and break anyone trying to use it. In addition, the version will remain the same when shipping versions stabilize.

Reply:
Ok. That template is also currently marked to not ship. So I think simply marking this as `non-shipping` and leaving the `IsPackable` as is resolves your concern.